### ISSUE TITLE
No compatible version found: babel-types@^6.1.10

### ISSUE THREAD

**Issue by SystemParadox** [babel/babel#3018]
Hi,

I tried to install my project this afternoon, only to discover that many babel packages are now depending on version 6.1.12 for various packages, but they have not been published on npm yet.

Please can this be fixed as a matter of urgency!

Is there something that can be done to ensure that npm is always updated in future?

Thanks.

**Comment by jaylaw81** [comment]
Same for babel-template
```bash
npm ERR! No compatible version found: babel-template@^6.1.12
npm ERR! Valid install targets:
npm ERR! ["6.0.16","6.0.15","6.0.14","6.0.12","6.0.2"]
```

**Comment by sebmck** [comment]
Apparently npm is weird and has different before for the npm tags `stable` and `latest`. The npm tags have been cleared and fixed as of 6.1.18 which caused versions to be picked up by npm that were still in progress/broken that should have been inaccessible.
